### ISSUE TITLE
Chemlib items need generic tags

### ISSUE THREAD

**Issue by HexFlareheart** [al132mc/chem-lib#77]
As a general rule, for most item types that use tags to designate what material they're made of, the broad tag for that item type is also present; a common zinc ingot is tagged not just with "forge:ingots/zinc", but also "forge:ingots". The latter tag is used not for recipe unification or anything like that, like the specific material subtag is (except in weird fringe cases like Industrial Foregoing's Meatallurgic Mycelial Generator), but can be and IS regularly used for automatically detecting item types for the purposes of sorting, automatically generating recipes, etc.

Consequently it's kind of a problem that every single item in Chemlib that uses tags doesn't use the generics. Every plate is only tagged with the specific forge:plates/[material] tag, every ingot is only tagged with the specific forge:ingots/[material] tag, every dust, etc. This can render tag filtering, an extremely powerful way of controlling item movement, completely useless when Chemlib items are in the mix, as not all means are going to identify that this item that has forge:dusts/zinc is, in fact, a dust; for all they know it's a material that's just intended to be subbed in for zinc dust in some situations.

So, basically, I'm here to request that the respective general tags for each item type be added to dusts, storage blocks, ingots, nuggets, and plates. This would make handling Chemlib items with automation much easier. I'd also recommend adding a couple generic tags like "chemlib:elements" and "chemlib:compounds" for differentiating between, well, elements and compounds. Less important, but still a good idea.

(also relatedly, please add aluminium tags to aluminum items; both are correct spellings, and no one is ever going to agree to stick with one or the other, so the best solution to the problem is to just tag stuff with both and call it a day.
